types/enums/UseClasses.ts

```diff
@@ -0,0 +1,14 @@
+export const GLAUseClasses = {
+  bTwo: 'B2 - General industry',
+  bEight: 'B8 - Storage and distribution',
+  cOne: 'C1 - Hotels',
+  cTwo: 'C2 - Residential institutions',
+  cTwoA: 'C2a - Secure residential institutions',
+  cThree: 'C3 - Dwellinghouses',
+  cFour: 'C4 - Houses in multiple occuplation (HMOs)',
```

Comment:
Spelling mistake: `occuplation` -> `occupation`

Reply:
Thanks for catching these, both updated!